cloudencrypt/log.go

```diff
@@ -0,0 +1,48 @@
+package cloudencrypt
+
+import (
+	"context"
+	"fmt"
+	"log"
+)
+
+// LogEncryptor wraps another Encryptor and adds logging.
+type LogEncryptor struct {
```

Comment:
```suggestion
type EncryptorWithLogger struct {
```
LogEncryptor seems to encrypt the logs.

Reply:
Maybe LoggingEncryptor then? I'd like to keep Encryptor as a suffix.

Reply:
Better but still not ideal. Ill leave that one on you just make sure someone will use it as encryptor for logs